buildpipeline/linux.arm64.groovy

```diff
@@ -0,0 +1,44 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Note that the parameters will be set as env variables so we cannot use names that conflict
+// with the engineering system parameter names.
+// CGroup - Build configuration.
+// TestOuter - If true, runs outerloop, if false runs just innerloop
+
+def submittedHelixJson = null
+
+simpleDockerNode('microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-arm64-a3ae44b-20180315221921') {
+    stage ('Checkout source') {
+        checkoutRepo()
+    }
+
+    def logFolder = getLogFolder()
+
+    stage ('Initialize tools') {
+        // Init tools
+        sh './init-tools.sh'
+    }
+    stage ('Generate version assets') {
+        // Generate the version assets.  Do we need to even do this for non-official builds?
+        sh "./build-managed.sh -- /t:GenerateVersionSourceFile /p:GenerateVersionSourceFile=true"
+    }
+    stage ('Sync') {
+        sh "./sync.sh -p -- /p:ArchGroup=arm64"
```

Comment:
This sync command is failing on arm64.  Presumably because the arm64 runtime is not available (because there are no corefx official builds yet)

Reply:
Removing

Reply:
For reference the failure was 
```
chmod: cannot access '/git/corefx/bin/testhost/netcoreapp-Linux-Debug-arm64/dotnet': 
       No such file or directory
./git/corefx/external/runtime/runtime.depproj(73,5): error MSB3073: 
     The command "chmod +x ./git/corefx/bin/testhost/netcoreapp-Linux-Debug-arm64/dotnet" 
     exited with code 1.

```